content/episodes/2019-02-18-S04E06-decentralize-it.md

```diff
@@ -0,0 +1,244 @@
+
+    **Manoush Z.:** Despite the risks, decentralized alternatives come with a lot of optimism baked in. We're gonna keep that in mind today as we focus on a few examples. We're gonna look at how decentralization could make social media better, how it could give us back control of our privacy and our data, how it could be used to protect journalism and information, maybe. And how it might even revamp the entire internet infrastructure as we know it.
+
+    **Eugen Rochko:** My name is Eugen Rochko, and I am the founder and developer of Mastodon, the decentralized social network.
+
+    **Manoush Z.:** Before Eugen Rochko created Mastodon, he loved Twitter.
+
+    **Eugen Rochko:** I was an avid Twitter user, and the format was close to my heart, but I was not happy with the direction the company was going.
+
+    **Manoush Z.:** So he wanted to build an alternative, and right away Eugen knew he wanted Mastodon to be decentralized.
+
+    **Eugen Rochko:** With a platform like Mastodon, when you have your own server, you make the rules, and nobody can change the rules for you.
+
+    **Manoush Z.:** Anyone can host a server, or node. In Mastodon speak, servers are called "instances." Eugen says this model forms a kind of federated universe, a fedeverse, and because of this it's much easier to deal with challenges that Twitter struggles to tackle.
+
+    **Eugen Rochko:** In a decentralized network, every admin can decide, "What do I want to host on my node? What do I not want to host on my node?" If you don't want Nazis, you just ban all Nazis from your node. If bad actors wanted to start a Mastodon server they could, but normal people could too. You can just say, "I don't wanna receive any messages from this bad server, there's bad people there."
+
+    **Manoush Z.:** Eugen says he counts about 1.8 million registered users, although he concedes that only about 170,000 of those users seem to be active on the platform, but it's a start. The other advantage he points to is that Mastodon doesn't depend on user data and ad targeting to make money.
+
+    **Eugen Rochko:** Many of the reasons for Twitter's decision making going against user's interests is driven by its centralization. They have to perform financially. Now with a decentralized approach, we don't really need to monetize because the costs of operating everything are just so comparatively small for any given independent entity, right?
+
+    **:** For me, it's like, let's say about $500 per month to host my server of Mastodon, and that's housing 300,000 registered accounts. I have a Patreon that more than covers that. For many other administrators of Mastodon servers, it's the same situation.
```

Comment:
Missing a name here.

Reply:
Though on further thought and looking at the rest of the transcript, in some sections the speaker name is repeated at the start of each paragraph when it's still the same speaker, and in other places the same speaker goes for a few paragraphs with their name only appearing on the first one. I'm not sure which way is more standard but we should be consistent, within a single episode at least. From past episodes it looks like we've usually done the name once when the speaker changes so maybe we should stick with that format here?

Reply:
I think it's a different speaker. I'll listen to the episode and see who is talking.

Reply:
Still Eugen.

Reply:
A few times in the first part of the transcript the speaker name is repeated for each paragraph when it's still the same speaker (I'm guessing this was transcribed in sections by different people). Nothing is broken and it's still understandable so not a real showstopper problem, just inconsistent. I'd be in favor of removing those repeated names in the first third of the transcript but it's up to you.